app/controllers/post_controller.rb

```diff
@@ -0,0 +1,6 @@
+class PostController < ApplicationController
+  def show
+    user_id = params[:id].to_i
+    @posts = Post.where(user_id == :author_id)
+  end
+end
```

Comment:
- [ ] The posts controller should also have both a `show` and an `index` method. Kindly aadd the `index` method as this is a requirement